71_RayTracingPipeline/app_resources/raytrace.rgen.hlsl

```diff
@@ -53,9 +55,11 @@ void main()
         rayDesc.TMin = 0.01;
         rayDesc.TMax = 10000.0;
         
+        [[vk::ext_storage_class(spv::StorageClassRayPayloadKHR)]]
         PrimaryPayload payload;
```

Comment:
same as https://github.com/Devsh-Graphics-Programming/Nabla-Examples-and-Tests/pull/212/files#r2280962435